rust/core-lib/src/config.rs

```diff
@@ -328,6 +328,12 @@ impl ConfigManager {
     pub fn default_buffer_config(&self) -> BufferConfig {
         self.get_buffer_config(None, None)
     }
+
+    /// Path to themes sub directory inside config directory
+    pub fn get_themes_dir(&self) -> PathBuf {
+        let themes_dir = self.config_dir.as_ref().unwrap().join("themes");
```

Comment:
there is no guarantee that `self.config_dir` exists, so we can't safely unwrap here.

Reply:
Ok, for this I can check for `config_dir` before calling this method or return a `Result` from here and then handle the error in the calling fn. I think the last method is better.